### DISCUSSION TITLE
My account is disabled

### DISCUSSION THREAD

**Discussion by Silenthhh** [community/community#130772] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Issues

### Body

What GitHub account is disabled

**Comment by HemalathaT2** [comment]
If your payment is past due because of billing problems, your account's paid features will be locked. You can unlock and access your account by updating your account's payment method and resuming paid status.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Silenthhh ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

The community is unable to assist with nor escalate your ticket and it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 
As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
